syntax = "proto3";
package rdma;

message NodeIDRequest {
    string ip = 1;
    string name = 2;
    uint32 node_type_enum = 3;
}

